### ISSUE TITLE
Some questions about iGPU performance and basic all other things

### ISSUE THREAD

**Issue by ahaadkabir** [yrambler2001/Hackintosh-Intel-i5-10600-iGPU-AsRock-B460M-Pro4#1]
Hi, can you please confirm that your hackintosh with intel UHD 630 graphics is fully working normally as expected and has no other major or minor issues, glitches or bugs.

**Comment by yrambler2001** [comment]
Hi @ahadrox it is working as expected, i don't use it for DRM video play so I don't know how to test DRM video play
However I am facing an issue with display init after startup on two 1080p monitors (the second monitor is black/random black lines), I fix this issue until reboot by selecting another resolution and then switch back to original resolution.

**Comment by yrambler2001** [comment]
I've added Geekbench 5 iGPU test to the readme.md https://browser.geekbench.com/v5/compute/1327011

**Comment by ahaadkabir** [comment]
Thank you very much for your response. So I see the graphics is working fully okay. And I've seen you are using a 480GB SSD. May I know intially after installing macOS, how much storage space is taken by the macOS itself? I'm planning to buy a 240GB NVMe SSD for the 1st build, so I'm bit worried about storage space, though I don’t need huge storage for now. But it would be great if you can answer that. Again, thanks.

**Comment by yrambler2001** [comment]
I think it will be around 25-40gb, Now I have around 80gb used with installed apps that I use.

**Comment by ahaadkabir** [comment]
Thank you very much again. I appreciate your timely replies. I'm closing this and I'll open again if I need your help again. Thanks.

**Comment by ahaadkabir** [comment]
Hi, just wanted to know if your motherboard has native NVRAM support or not. I saw SSDT-PMC on the EFI so does that mean that it is natively supported now? And apart from that, I came to know that my ASRock motherboard is not B460M Pro4, rather it’s a full sized ATX B460 Pro4 motherboard.  Do you think your EFI would work for me or should I change my motherboard?

**Comment by yrambler2001** [comment]
Hi, I think the EFI will work for your MB,
https://www.bilibili.com/read/cv6737693/
> 4. SSDT-PMC.aml enables NVRAM (Z370, Z490 and B460 native support, this patch is not required, other motherboards basically need) optional

There is not much info about hackintosh on the 4xx series intel chipest, looks like B460 has native nvram support and this patch is not required, I have not tested it without the SSDT-PMC.

**Comment by ahaadkabir** [comment]
Thanks, than I guess I'll try without the nvram patch to see if it actually supported natively or not. I'll let you know when I'm done.

**Comment by ahaadkabir** [comment]
Hi, after a lot of trying I was successful to install macOS Catalina 10.15.7 . But I faced issues with USB 3 ports. I used the same USBInjectAll and XHCI-Unsupported kext from your EFI but it did not detected all the USB 3 ports. Therefore I could not map the USB ports at all. But in the SSDT side, I just used SSDT-PLUG, SSDT-EC-USBX and SSDT-AWAC only. Native NVRAM is 100% working. OpenCore was latest 0.6.2 release version. Just want you to let you know that I have the exact motherboard as yours but processor is different (i5 10400).
